gardening/demo/res_partner.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="res_partner_7" model="res.partner">
+      <field name="name">Soil Vendor</field>
+      <field name="street">Soily</field>
+      <field name="country_id" ref="base.cl"/>
+      <field name="city">Soil</field>
+      <field name="is_company" eval="True"/>
+      <field name="state_id" ref="base.state_cl_01"/>
+      <field name="category_id" eval="[(6, 0, [])]"/>
+    </record>
+  <record id="res_partner_8" model="res.partner">
+    <field name="name">Jane Graden</field>
+    <field name="commercial_company_name">Jane Graden</field>
+    <field name="phone">+32548979443</field>
+    <field name="is_company" eval="True"/>
+    <field name="category_id" eval="[(6, 0, [])]"/>
+    <field name="email">janes.graden@example.com</field>
+  </record>
+  <record id="res_partner_9" model="res.partner">
+    <field name="name">Jhon Gredan</field>
+    <field name="commercial_company_name">Jhon Gredan</field>
+    <field name="phone">+32 89764983</field>
+    <field name="is_company" eval="True"/>
+    <field name="category_id" eval="[(6, 0, [])]"/>
+    <field name="email">jhon.gredan@example.com</field>
+  </record>
+  <record id="res_partner_10" model="res.partner">
+    <field name="name">Alain Garden</field>
```

Comment:
Isn't it automatically created at an employee's creation?

Reply:
What are you referring to ?